Mage.Client/release/sample-decks/Commander/Commander 2020/Arcane Maelstrom.dck

```diff
@@ -0,0 +1,93 @@
+1 [C20:206] Crackling Drake
+1 [C20:161] Surreal Memoir
+1 [C20:32] Decoy Gambit
+1 [C20:50] Deflecting Swat
+1 [C20:59] Curious Herd
+1 [C20:146] Chaos Warp
+1 [C20:148] Comet Storm
+1 [C20:149] Commune with Lava
+1 [C20:160] Starstorm
+1 [C20:192] Strength of the Tajuru
+1 [C20:203] Artifact Mutation
+1 [C20:227] Prophetic Bolt
+1 [IKO:111] Clash of Titans
+1 [IKO:180] Channeled Force
+1 [C20:108] Chemister's Insight
+1 [C20:111] Frantic Search
+1 [C20:127] Whiplash Trap
+1 [C20:169] Crop Rotation
+1 [C20:171] Evolution Charm
+1 [C20:174] Harrow
+1 [C20:178] Hunter's Insight
+1 [C20:179] Hunting Pack
+1 [C20:184] Natural Connection
+1 [C20:190] Slice in Twain
+1 [C20:193] Tribute to the Wild
+1 [C20:216] Growth Spiral
+1 [C20:230] Temur Charm
+1 [C20:53] Lavabrink Floodgates
+1 [C20:70] Twinning Staff
+1 [C20:67] Bonder's Ornament
+1 [C20:237] Arcane Signet
+1 [C20:240] Commander's Sphere
+1 [C20:245] Lightning Greaves
+1 [C20:252] Sol Ring
+1 [C20:124] Swarm Intelligence
+1 [IKO:200] Primal Empathy
+1 [C20:62] Predatory Impetus
+1 [C20:37] Psychic Impetus
+1 [C20:55] Shiny Impetus
+1 [C20:196] Wilderness Reclamation
+1 [C20:263] Cinder Glade
+1 [C20:269] Desolate Lighthouse
+1 [C20:273] Exotic Orchard
+1 [C20:284] Kessig Wolf Run
+1 [C20:290] Mossfire Valley
+1 [C20:291] Mosswort Bridge
+1 [C20:296] Oran-Rief, the Vastwood
+1 [C20:306] Scavenger Grounds
+1 [C20:322] Yavimaya Coast
+1 [C20:264] Command Tower
+1 [C20:275] Frontier Bivouac
+1 [C20:279] Gruul Turf
+1 [C20:280] Halimar Depths
+1 [C20:283] Izzet Boilerworks
+1 [C20:292] Myriad Landscape
+1 [C20:304] Rupture Spire
+1 [C20:311] Simic Growth Chamber
+1 [IKO:252] Rugged Highlands
+1 [IKO:255] Swiftwater Cliffs
+1 [IKO:256] Thornwood Falls
+5 [IKO:274] Forest
+2 [IKO:273] Forest
+1 [IKO:272] Forest
+3 [IKO:265] Island
+1 [IKO:263] Island
+1 [IKO:264] Island
+2 [IKO:269] Mountain
+2 [IKO:270] Mountain
+1 [IKO:271] Mountain
+SB: 1 [C20:9] Kalamax, the Stormsire
+LAYOUT MAIN:(2,8)(CMC,true,70)|()()([C20:36])([C20:2],[C20:60],[C20:150])([C20:33],[C20:115],[C20:120],[C20:125],[C20:229],[C20:253],[C20:118],[C20:206])([C20:18],[C20:13],[C20:153])([C20:147],[C20:151],[C20:220],[C20:234])([C20:63],[C20:210])([C20:263],[C20:269],[C20:273],[C20:284],[C20:290],[C20:291],[C20:296],[C20:306],[C20:322],[C20:264],[C20:275],[C20:279],[C20:280],[C20:283],[C20:292],[C20:304],[C20:311],[IKO:252],[IKO:255],[IKO:256],[IKO:274],[IKO:274],[IKO:274],[IKO:273],[IKO:274],[IKO:274],[IKO:273],[IKO:272],[IKO:265],[IKO:263],[IKO:264],[IKO:265],[IKO:265],[IKO:269],[IKO:269],[IKO:270],[IKO:270],[IKO:271])([C20:169],[C20:252])([C20:148],[C20:149],[C20:160],[C20:192],[C20:203],[C20:171],[C20:193],[C20:216],[C20:237],[C20:245])([C20:32],[C20:50],[C20:146],[C20:111],[C20:174],[C20:178],[C20:184],[C20:230],[C20:70],[C20:67],[C20:240],[IKO:200],[C20:37],[C20:55])([C20:114],[C20:161],[C20:59],[IKO:180],[C20:108],[C20:190],[C20:53],[C20:196])([C20:227],[IKO:111],[C20:127],[C20:62])()([C20:179],[C20:124])
```

Comment:
GUI layout settings must be removed from sample decks (all lines started with `LAYOUT`).